deeppavlov/models/syntax_parser/parser.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+import numpy as np
+
+from dependency_decoding import chu_liu_edmonds
+
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.models.component import Component
+
+
+@register('chu_liu_edmonds_transformer')
+class ChuLiuEdmonds(Component):
+    """
+    A wrapper for Chu-Liu-Edmonds algorithm for maximum spanning tree
+    """
+    def __init__(self, min_edge_prob=1e-6, **kwargs):
+        self.min_edge_prob = min_edge_prob
+
+    def __call__(self, probs: List[np.ndarray]) -> List[List[int]]:
+        """Applies Chu-Liu-Edmonds algorithm to the matrix of head probabilities.
+
+        probs: a 3D-array of probabilities of shape B*L*(L+1)
```

Comment:
Please specify the documentation for the output: purpose, format and/or optionally example of output.